pinot-broker/src/main/java/org/apache/pinot/broker/broker/BrokerAdminApiApplication.java

```diff
@@ -72,7 +69,7 @@ private void setupSwagger() {
     beanConfig.setContact("https://github.com/apache/incubator-pinot");
     beanConfig.setVersion("1.0");
     beanConfig.setSchemes(new String[]{CommonConstants.HTTP_PROTOCOL, CommonConstants.HTTPS_PROTOCOL});
-    beanConfig.setBasePath(_baseUri.getPath());
+    beanConfig.setBasePath("/");
```

Comment:
I am not sure if this is related to TLS or not.  What breaks if this change is not done?

Reply:
baseUri is a useless variable. the path is always set to "/" (see above in prev version, controller uses "/" directly too).